setup_cfg_fmt.py

```diff
@@ -103,7 +103,12 @@ def cb(match: Match[str]) -> str:
 def _first_file(setup_cfg: str, prefix: str) -> str | None:
     prefix = _case_insensitive_glob(prefix)
     path = _adjacent_filename(setup_cfg, prefix)
-    for filename in sorted(glob.iglob(f'{path}*')):
+
+    # make asciidoc last
```

Comment:
better would be to explain _why_ rather than _what_.  something like "prefer non-asciidoc because pypi does not render it"